Sources/SwiftProtobuf/TextFormatEncodingVisitor.swift

```diff
@@ -611,9 +611,10 @@ internal struct TextFormatEncodingVisitor: Visitor {
   private mutating func _visitMap<K, V>(
     map: Dictionary<K, V>,
     fieldNumber: Int,
+    lessThanFunc: (K, K) -> Bool,
```

Comment:
Nit: `isOrderedBefore` would be a cleaner name that matches the argument name used by Swift's stdlib for other sorting predicates.

Reply:
done.